src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```diff
@@ -494,9 +497,28 @@ compiler.err.same.binary.name=\
 compiler.err.duplicate.case.label=\
     duplicate case label
 
+compiler.err.pattern.dominated=\
+    this pattern is dominated by a preceding pattern
```

Comment:
Not sure whether the concept of "dominance" is the one that will work best here. As I said elsewhere, this is, morally, unreachable code.

Reply:
The specification uses "dominance", so it seemed appropriate to use the same term that is used by the specification. We can say "unreachable code", of course, but it will not be consistent with what the specification says, and also with what we do for duplicate constant labels. Also considering code like:
```
switch (o) {
     case A a -> {}
     case B b -> {} //error on this line
}
```

It may not be obvious why the code is "unreachable", while saying "dominated" might be more helpful/clear.